src/styles/_common.scss

```diff
@@ -160,6 +160,7 @@ button {
         width: 100%;
         height: 400px;
         background: rgba(0,0,0,0.7);
+        z-index: 1;
```

Comment:
Good thing you don't do `z-index: 999999`. That would look quite amateur. Which you are not! ;)